src/server/boot/03-audit.js

```diff
@@ -0,0 +1,38 @@
+import optionalBasicAuth from '../middleware/optional-basic-auth'
+import { models } from '../models'
+import { audit, getClientData } from '../util/audit'
+
+export default function (app) {
+  app.get(
+    '/api/audit-events',
+    optionalBasicAuth(),
+    app.requirePermission('view audit log'),
+    app.wrap(async (req, res) => {
+      const filter = JSON.parse(req.query.filter || '{}')
+
+      const where = filter.where || {}
+      const limit = +filter.limit || 250
```

Comment:
just noticed this now, using `+` to coerce strings into numbers is not something i personally like, it is quite unexpected in most places. We should be improving the way these params are parsed, but for now just using `Number(filter.limit)` would be better here, and similarly on the line below